src/OpenLoco/src/Ui/Windows/MusicSelection.cpp

```diff
@@ -21,6 +21,8 @@ namespace OpenLoco::Ui::Windows::MusicSelection
 
     static constexpr uint8_t kRowHeight = 12; // CJK: 15
 
+    static std::vector<uint8_t> _musicTracks;
```

Comment:
I'm not a fan of making this static, but I can see you've done it because the mouse scroll handler is static. Having a more OOP-style approach to these windows is what we're trying to do, so is it possible to make the mouse scroll handler not static?

Reply:
I think I wanted it to be an attribute of a music selection window object, so making it static was my quick solution considering it doesn't appear to be an object/class currently (and I'm not sure I'm confident enough to personally rewrite an entire file to be object oriented yet lol)